features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/actionlist/model/TimelineItemAction.kt

```diff
@@ -40,6 +40,7 @@ sealed class TimelineItemAction(
     data object ReportContent : TimelineItemAction(CommonStrings.action_report_content, CompoundDrawables.ic_compound_chat_problem, destructive = true)
     data object EndPoll : TimelineItemAction(CommonStrings.action_end_poll, CompoundDrawables.ic_compound_polls_end)
     data object Pin : TimelineItemAction(CommonStrings.action_pin, CompoundDrawables.ic_compound_pin)
-    //TODO use the Unpin compound icon when available.
+
+    // TODO use the Unpin compound icon when available.
```

Comment:
Are you sure about removing this TODO? The unpin icon is already in https://github.com/element-hq/compound-design-tokens, we just need to import the new icons in https://github.com/element-hq/compound-android and release a new version.

Reply:
Or maybe not 🫠 https://github.com/element-hq/compound-design-tokens/pull/104